WordPress/Classes/ViewRelated/Reader/ReaderPostsViewController.m

```diff
@@ -870,8 +840,13 @@ - (void)cleanupAfterRefresh
 
 - (void)tableViewHandlerWillRefreshTableViewPreservingOffset:(WPTableViewHandler *)tableViewHandler
 {
-    [UIView performWithoutAnimation:^{
-        [self cleanupAfterRefresh];
+    [[self managedObjectContext] performBlockAndWait:^{
```

Comment:
Sorry about the extra noise, just wanna re-re-re check. Were you able to check if the `performBlockAndWait` is 100% required here? (Thanks!)